example/material.json

```diff
@@ -63,15 +64,7 @@
         {
             "element": "Si",
             "name": "elemental_ratio",
-            "value": 0.71
-        },
-        {
-            "name": "inchi",
```

Comment:
inchi doesn't really apply to materials, in fact we don't calculate it for them.